artemis/config.py

```diff
@@ -611,6 +611,29 @@ class DomainExpirationScanner:
                 int, "The scanner warns if the domain's expiration date falls within this time frame from now."
             ] = get_config("DOMAIN_EXPIRATION_TIMEFRAME_DAYS", default=30, cast=int)
 
+        class SqlInjectionDetector:
+            SQL_INJECTION_TIME_THRESHOLD: Annotated[
+                int,
+                "Seconds to method sleep() or pg_sleep()",
+            ] = get_config("THRESHOLD", default=2, cast=int)
+
+            SQL_INJECTION_SLEEP_PAYLOADS: Annotated[
+                List[str],
+                "List of payloads with sleep() and pg_sleep() functions.",
+            ] = get_config(
+                "SLEEP_PAYLOAD",
+                default=[
+                    f"'||sleep({SQL_INJECTION_TIME_THRESHOLD})||'",
```

Comment:
IMO this doesn't have to be a config variable but part of the sql injection module logic